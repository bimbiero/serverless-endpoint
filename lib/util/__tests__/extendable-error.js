'use strict';

/* global describe jest test expect */
jest.dontMock('../extendable-error');

const ExtendableError = require('../extendable-error');

describe('util.extendable-error.js', () => {
  test('constructor', () => {
    function extendableError() {
      throw new ExtendableError();
    }

    expect(extendableError).toThrowErrorMatchingSnapshot();
  });
});
