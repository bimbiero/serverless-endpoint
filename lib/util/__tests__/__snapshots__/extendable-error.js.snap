// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util.extendable-error.js constructor 1`] = `""`;
